src/Inventory.java

```diff
@@ -0,0 +1,153 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+/**
+ * Created by Nigel on 6/16/16.
+ */
+public class Inventory {
+
+    private ArrayList<Item> items = new ArrayList<>();
+    Scanner scanner = new Scanner(System.in);
+
+    public void addItem(int id) {
+        scanner.reset();
+        Item theItem = new Item();
+        System.out.println("Please enter the new Item's Name: ");
+        theItem.setName(scanner.nextLine());
+        System.out.println("How many Items are there? ");
+        theItem.setQuantity(scanner.nextInt());
+        System.out.println("What is the price of the Item? ");
+        theItem.setPrice(scanner.nextInt());
+        theItem.setId(id);
+        items.add(theItem);
+    }
+
+    public boolean isListEmpty() {
```

Comment:
Not a great name.  Which list? What is in the list? a more appropriate name is `isItemsEmpty`